WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1087,7 +1092,10 @@ decisions = {
 		allow = {
 			war = no
 			NOT = { trait = incapable }
-			completely_controls = d_aragon
+			completely_controls = c_alto_aragon
+			completely_controls = c_zaragoza
+			completely_controls = c_albarracin
+			completely_controls = c_calatayud
```

Comment:
It would, hence why I suggested it. The AI shouldn't be allowed to take this at start, otherwise Aragon could form in the 5th century. always = no is the quick fix.